cmd/serf/command/agent/rpc_client_test.go

```diff
@@ -112,6 +112,59 @@ WAIT:
 	}
 }
 
+func TestRPCClientForceLeave_prune(t *testing.T) {
+	client, a1, ipc := testRPCClient(t)
+	a2 := testAgent(nil)
+	defer ipc.Shutdown()
+	defer client.Close()
+	defer a1.Shutdown()
+	defer a2.Shutdown()
+
+	if err := a1.Start(); err != nil {
+		t.Fatalf("err: %s", err)
+	}
```

Comment:
I think the `defer a1.Shutdown()` should be moved to after this startup block. I have no idea if you can shutdown a client that didn't start.